robottelo/hosts.py

```diff
@@ -45,6 +45,7 @@
 }
 
 
+@cached_property
```

Comment:
you would actually use the `@cached` decorator here. `cached_property` is for use in classes. However, I am going to recommend against caching.